examples/splunk-sink/README.md

```diff
@@ -0,0 +1,62 @@
+# Splunk sink example
+
+The solution helps you set up a log-streaming pipeline from Stackdriver Logging to Splunk.
+
+The example is for a project-level sink, but it can be easily be adapted for aggregated log sinks.
+
+## Instructions
+
+1. Fill the required variables in the `example.auto.tfvars` file located in the `examples/` directory.
```

Comment:
We'll need to update this path if we make the suggested changes around renaming files and providing a sample file rather than an active `tfvars`.

Resolve this comment if this comment is not applicable.